frappe/model/db_schema.py

```diff
@@ -45,6 +45,7 @@ class InvalidColumnName(frappe.ValidationError): pass
 	,'Signature':	('longtext', '')
 	,'Color':		('varchar', varchar_len)
 	,'Barcode':		('longtext', '')
+	,'Geolocation':	('longtext', '')
```

Comment:
fix formatting

Reply:
some how sublime is not showing any funny formatting?!
![screenshot from 2017-10-25 13-51-40](https://user-images.githubusercontent.com/1040161/31988128-a9d63b02-b98b-11e7-9bbd-12819376e88a.png)

I'll add extra tab to all?